packages/roosterjs-editor-api/lib/utils/applyInlineStyle.ts

```diff
@@ -27,7 +42,7 @@ export default function applyInlineStyle(
                 (getTagOfNode(node.firstChild) == 'BR' && !node.firstChild.nextSibling));
         if (isEmptySpan) {
             editor.addUndoSnapshot();
-            callback(node as HTMLElement);
+            entitySafeCallback(node as HTMLElement);

```

Comment:
I think we only need to check if this node is read only (node.isContentEditable), no need to check entity.

Reply:
Do you mean this for all the cases, or just the empty `<span>`?

Reply:
for all the cases

Reply:
Yup, you were right 😁